Robust/View/MainWindow.xaml.cs

```diff
@@ -19,5 +20,8 @@ public partial class MainWindow : Window
     public MainWindow()
     {
         InitializeComponent();
+
+        ProductViewModel pvm = new ProductViewModel();
+        DataContext = pvm;
```

Comment:
Da vi ikke bruger vores pvm til noget kan vi bare sige

```suggestion
        DataContext = new ProductViewModel();
```